### ISSUE TITLE
Image Alt text/tags not visible with Brizy Page Builder

### ISSUE THREAD

**Issue by kevinpauls** [ThemeFuse/Brizy#630]
When a page i created with Brizy Page Builder the image alt text input in Wordpress does not show in published page.
